src/univariate/discrete/dirac.jl

```diff
@@ -0,0 +1,30 @@
+"""
+    Dirac(value)
+
+A *Dirac distribution* is parametrized by its only value, and takes its value with probability 1.
+
+```julia
+d = Dirac(2.)  # Dirac distribution with value = 2.
+rand(d)       # Always returns the same value
```

Comment:
Again, punctuation, and the comment isn't aligned with the one above (which I assume is what you wanted).